[DOCS] Osquery features in 8.5 (#2561)
elastic/security-docs: docs/detections/invest-guide-run-osquery.asciidoc

[[invest-guide-run-osquery]]
=== Run Osquery from investigation guides

> Investigations guides can be used with rules and provide guidance for triaging, analyzing, and responding to potential security issues. When you build a custom rule, you can also set up an investigation guide that incorporates Osquery. This allows you to run live queries from a rule's investigation guide as you analyze alerts produced by the rule.

Reviewer: ```suggestion
Investigation guides on detection rules provide guidance for triaging, analyzing, and responding to potential security issues. When you build a custom rule, you can also set up an investigation guide that incorporates Osquery. This allows you to run live queries from a rule's investigation guide as you analyze alerts produced by the rule.
```

[role="screenshot"]
image::images/osquery-investigation-guide.png[Shows a live query in an investigation guide]

> .Requirements

Reviewer: Maybe move the reqs above the screenshot -- they kind of get buried under the image?

[sidebar]
--
* The {kibana-ref}/manage-osquery-integration.html[Osquery manager integration] must be installed.
* {agent}'s {fleet-guide}/view-elastic-agent-status.html[status] must be `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
* Your role must have {kibana-ref}/osquery.html[Osquery feature privileges].
--

[float]
[[add-live-queries-ig]]
=== Add live queries to an investigation guide

NOTE: You can only add Osquery to investigation guides for custom rules because prebuilt rules cannot be edited.

> Go to Rules page, then do the following:
> 
> . Select a rule, then click *Edit rule settings* on the rule details page.

Reviewer: The lead-in seems like a step in itself, but it could also just bundle them all together:
```suggestion
. Go to *Manage* -> *Rules*, select a rule, then click *Edit rule settings* on the rule details page.
```

> . Go to the *About* tab, then expand the rule's advanced settings.

Reviewer: Going to a tab sounds odd; either click or select:
```suggestion
. Select the *About* tab, then expand the rule's advanced settings.
```
